docs/other-macros.md

```diff
@@ -59,7 +59,7 @@ TEST_CASE( "SUCCEED showcase" ) {
 }
 ```
 
-* `STATIC_REQUIRE`
+* `STATIC_REQUIRE` and `STATIC_CHECK`
 
 > [Introduced](https://github.com/catchorg/Catch2/issues/1362) in Catch2 2.4.2.
```

Comment:
This should probably be edited to be along the lines of "`STATIC_REQUIRE` was introduced ..." and then having a "`STATIC_CHECK` was introduced in ..." separately.

Reply:
Done.